docs/docs/examples/authz.md

```diff
@@ -0,0 +1,290 @@
+
+        allow_request = true :- not seccomp_unconfined
+        EOF
+
+    The plugin is watching the policy definition file for changes. Each time we change the file, the plugin reads the file and sends it to OPA. To manually send the policy to OPA, you can use the following API:
+
+        curl -X PUT --data-binary @example.rego \
+            http://localhost:8181/v1/policies/example_policy
+
+    This API is idempotent so sending the policy multiple times is fine. Go ahead and try it yourself.
+
+1. Test the policy is working by running a simple container:
+
+        docker run hello-world
+
+    Now try running the same container but disable seccomp (which should be prevented by the policy):
+
+        docker run --security-opt seccomp=unconfined hello-world
+
+    When Docker processes the run command it contacts the plugin to see if the request(s) should be allowed. The plugin takes the request and executes a query against OPA using the request as input data to the query. The same API call that the plugin makes can be executed using curl:
+
+        curl -v -G http://localhost:8181/v1/data/opa/example/allow_request \
+            --data-urlencode \
+            'global=request:{"Body":{"HostConfig":{"SecurityOpt":["seccomp=unconfined"]}}}'
+
+    Because the document generated by the allow_request rule is undefined in this case, OPA responds with a 404.
+
+    You can re-run the same query with the default seccomp profile and see that it succeeds:
+
+        curl -G http://localhost:8181/v1/data/opa/example/allow_request \
+            --data-urlencode \
+            'global=request:{"Body":{"HostConfig":{"SecurityOpt":["seccomp=default"]}}}'
+
+    Congratulations! You have successfully prevented containers from running without seccomp!
+
+    So far, the policy has been defined in terms of input data from the plugin. In many cases, it's necessary to write policies against multiple data sources.
+
+    The rest of the example shows how you can grant fine grained access to specific clients. To do so, we will insert fake user data into OPA to simulate an authentication system.
+
+1. Create a Docker configuration file to include an extra HTTP header in all requests:
+
+        mkdir -p ~/.docker
+
+        # Backup your existing Docker configuration, just in case. You can replace the configuration
+        # which we generate below after you are done the example.
+        cp ~/.docker/config.json ~/.docker/config.json~
+
+        cat >~/.docker/config.json <<EOF
+        {
+            "HttpHeaders": {
+                "Authz-User": "bob"
+            }
+        }
+        EOF
+
+    Currently, Docker does not provide a way to authenticate clients. In Docker 1.12, clients can be authenticated using TLS
+    and there are plans to include other means of authentication. For the purpose of this example, we assume that an authentication
+    system is place.
+
+1. Insert user data into OPA:
```

Comment:
Could we explain that we're adding this data, which could either come from an external system or as part of policy.